x-pack/test/api_integration/deployment_agnostic/apis/observability/slo/find_slo.ts

```diff
@@ -28,80 +28,144 @@ export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
-          .send();
-
-        expect(response.body.results).length(2);
-        expect(response.body.page).eql(1);
-        expect(response.body.perPage).eql(333);
-        expect(response.body.total).eql(2);
-
-        response = await supertestWithoutAuth
-          .get(`/api/observability/slos`)
-          .query({ size: 222, kqlQuery: 'slo.name:irrelevant' })
-          .set(adminRoleAuthc.apiKeyHeader)
-          .set(internalHeaders)
-          .send()
-          .expect(200);
-
-        expect(response.body.page).eql(1); // always return page with default value
-        expect(response.body.perPage).eql(25); // always return perPage with default value
-        expect(response.body.size).eql(222);
-        expect(response.body.searchAfter).ok();
-        expect(response.body.results).length(1);
-        expect(response.body.results[0].id).eql(sloId2);
-
-        response = await supertestWithoutAuth
-          .get(`/api/observability/slos`)
-          .query({ kqlQuery: 'slo.name:integration' })
-          .set(adminRoleAuthc.apiKeyHeader)
-          .set(internalHeaders)
-          .send()
-          .expect(200);
-
-        expect(response.body.results).length(1);
-        expect(response.body.results[0].id).eql(sloId1);
-
-        return true;
+    describe('Find outdated SLOs', function () {
+      before(async () => {
+        adminRoleAuthc = await samlAuth.createM2mApiKeyWithRoleScope('admin');
+        internalHeaders = samlAuth.getInternalRequestHeader();
+
+        await generate({ client: esClient, config: DATA_FORGE_CONFIG, logger });
+
+        await dataViewApi.create({
+          roleAuthc: adminRoleAuthc,
+          name: DATA_VIEW,
+          id: DATA_VIEW_ID,
+          title: DATA_VIEW,
+        });
+
+        await sloApi.deleteAllSLOs(adminRoleAuthc);
+      });
+
+      after(async () => {
+        await dataViewApi.delete({ roleAuthc: adminRoleAuthc, id: DATA_VIEW_ID });
+        await cleanup({ client: esClient, config: DATA_FORGE_CONFIG, logger });
+        await sloApi.deleteAllSLOs(adminRoleAuthc);
+        await samlAuth.invalidateM2mApiKeyWithRoleScope(adminRoleAuthc);
+      });
+
+      it('finds outdated SLOs', async () => {
+        const outdatedSLO = {
+          ...DEFAULT_SLO,
+          name: 'outdated slo',
+        };
+        const recentSLO = {
+          ...DEFAULT_SLO,
+          name: 'recent slo',
+        };
+        const outdatedResponse = await sloApi.create(outdatedSLO, adminRoleAuthc);
+        const recentResponse = await sloApi.create(recentSLO, adminRoleAuthc);
+        const { id: outdatedSloId } = outdatedResponse;
+        const SOResponse = await sloApi.getSavedObject(adminRoleAuthc, outdatedSloId);
+        const savedObject = SOResponse.saved_objects[0];
+        const savedSlo = savedObject.attributes;
+        savedSlo.version = 1;
+        savedObject.attributes = savedSlo;
+
+        await sloApi.updateSavedObject(adminRoleAuthc, savedSlo, savedObject.id);
```

Comment:
You can remove some extra variable and update inline
```suggestion
        const savedObject = SOResponse.saved_objects[0];
        await sloApi.updateSavedObject(adminRoleAuthc,{ ...savedObject.attributes, version: 1 }, savedObject.id);
```